base_suspend_security/tests/test_base_suspend_security.py

```diff
@@ -30,3 +30,25 @@ def test_base_suspend_security(self):
         # this tests if _normalize_args conversion works
         self.env['res.users'].browse(
             self.env['res.users'].suspend_security().env.uid)
+
+    def test_suspend_security_on_search(self):
+        user_without_access = self.env["res.users"].create(
+            dict(
+                name="Testing Suspend Security",
+                login="nogroups",
+                email="nogroups@suspendsecurity.com",
+                groups_id=[(5,)],
+            )
+        )
+        model_ids = (
+            self.env["ir.model.access"]
+            .search([("group_id", "=", False)])
+            .mapped("model_id.id")
+        )
+        model = self.env["ir.model"].search([("id", "not in", model_ids)], limit=1)
```

Comment:
I find this a little convoluted. Why not pick some specific model from the base module of which you know it's not accessible?